src/OrchardCore.Modules/OrchardCore.Admin/Controllers/AdminController.cs

```diff
@@ -1,19 +1,25 @@
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using OrchardCore.DisplayManagement;
 
 namespace OrchardCore.Admin.Controllers
 {
     [Authorize]
     public class AdminController : Controller
     {
-        public AdminController()
+          private readonly IShapeFactory _shapeFactory;
+          
+        public AdminController(IShapeFactory shapeFactory)
         {
-
+            _shapeFactory = shapeFactory;
         }
 
-        public IActionResult Index()
+         public async Task<IActionResult> Index()
         {
-            return View();
+            var adminshape = await _shapeFactory.CreateAsync("AdminContent");
```

Comment:
I think you should do this directly from the view.

`@await Display.AdminContent()`